src/interpolation/ArborX_InterpMovingLeastSquares.hpp

```diff
@@ -0,0 +1,278 @@
+/****************************************************************************
+ * Copyright (c) 2023 by the ArborX authors                                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_INTERP_MOVING_LEAST_SQUARES_HPP
+#define ARBORX_INTERP_MOVING_LEAST_SQUARES_HPP
+
+#include <ArborX_AccessTraits.hpp>
+#include <ArborX_DetailsKokkosExtScopedProfileRegion.hpp>
```

Comment:
```suggestion
#include <Kokkos_Profiling_ScopedRegion.hpp>
```
That header was removed in https://github.com/arborx/arborx/commit/54036fba39b6a584128cbfc4d40804fc6af9474b.

Reply:
Changed locally